tools/c7n_gcp/development.md

```diff
@@ -0,0 +1,33 @@
+# Local development
```

Comment:
this likely should be copied into the docs, standalone md files won't get published.

Reply:
I was just wanting something in tree for local development. Happy to also have it in other places. Do symlinks work?

Reply:
Never tried re symlinks, the docs/makefile has file copies.